util_scripts/upload_synthetic_data/upload_synthetic_data.py

```diff
@@ -19,8 +19,6 @@
 
```

Comment:
Not a review comment, just a curiosity question: What does this file actually do?